README.md

```diff
@@ -83,12 +104,65 @@ the koji certificates directory (__~/.local/share/osbs-box/certificates/koji/__
 There is one for each koji user (by default, only _kojiadmin_ and _kojiosbs_ are users, but
 logging in creates a user automatically).
 
-#### Koji CLI (local)
+### Koji CLI (local)
 
 Coming soon<sup>TM</sup>
 
+### Docker registry
+
+OSBS-Box deploys a docker registry container for you. It is used both as the source registry for
+base images and the output registry for built images.
+
+You can access it just like any other docker registry using its external route. You can find the
+URL in the console or with `oc get route osbs-registry`.
+
+Since the certificate used for the registry is signed by our own, untrusted CA, the registry is
+considered insecure. To access the registry with various tools you need to:
+
+* `docker`: Add the registry URL to `insecure-registries` in __/etc/docker/daemon.json__
+            and restart docker
+* `podman`: Use the `--tls-verify=false` option
+* `skopeo`: Use the `--tls-verify=false` option (or `--(src|dest)-tls-verify=false` for copying)
+
+### Skopeo-lite
+
+Regular `skopeo` does not copy manifest lists. Builds may work even with base images copied using
+`skopeo`, but they will not use OSBS features related to manifest lists.
+
+For this purpose, OSBS-Box provides a `skopeo-lite` image.
+
+Use it with `podman`:
+
+```bash
+$ podman build skopeo-lite/ --tag skopeo-lite
```

Comment:
I think we can make this easier for people and automatically builds that image in https://quay.io/organization/containerbuildsystem namespace

Reply:
It appears that I do not have any of the permissions needed to do that, best I can do is create a repo in my quay.io namespace and build it from my forked osbs-box repo.